sql/moz-fx-data-shared-prod/mdn_yari_derived/dataset_metadata.yaml

```diff
@@ -0,0 +1,12 @@
+friendly_name: MDN Yari Dataset
```

Comment:
Why is dataset metadata needed here? Can we omit it to avoid creation of an empty dataset?

Reply:
Removed the dataset_metadata.yaml file.